test/e2e/mock-e2e-allowlist.js

```diff
@@ -112,12 +92,6 @@ const ALLOWLISTED_URLS = [
   'https://nft.api.cx.metamask.io/collections?chainId=0x539&contract=0x5fbdb2315678afecb367f032d93f642f64180aa3',
   'https://nft.api.cx.metamask.io/users/0x5cfe73b6021e818b776b421b1c4db2474086a7e1/tokens?chainIds=1&limit=50&includeTopBid=true&continuation=',
   'https://o1377931.ingest.sentry.io/api/6689755/envelope/?sentry_key=be397d53390141cda058e18f3749c8e4&sentry_version=7&sentry_client=sentry.javascript.react%2F7.102.1',
-  'https://on-ramp-content.api.cx.metamask.io/regions/networks?context=extension',
-  'https://on-ramp.api.cx.metamask.io/eligibility/mm-card?id=',
-  'https://on-ramp.api.cx.metamask.io/eligibility/mm-card?id=0x01a92b9df8ce522a8f98161a97378bfdb2cce64f2d4293657e7eefe818d9ffbc',
-  'https://on-ramp.api.cx.metamask.io/eligibility/mm-card?id=0xb1b1aa3036452bf32c1159266d1886279e4890ba907ed275e39fbc56906a8be2',
-  'https://on-ramp.api.cx.metamask.io/eligibility/mm-card?id=7ea301c0-6a98-11ef-a19e-f3acaa606eca',
-  'https://on-ramp.api.cx.metamask.io/eligibility/mm-card?id=8af26790-6a98-11ef-bee7-5553e37c0097',
```

Comment:
:information_source: 1 mock added for all the eligibility requests, no matter which id used